src/openrct2/platform/Crash.cpp

```diff
@@ -186,8 +187,9 @@ static bool OnCrash(
         exporter->Export(saveFilePathUTF8.c_str());
         savedGameDumped = true;
     }
-    catch (const std::exception&)
+    catch (const std::exception& e)
     {
+        printf("Failed to export save. Error: %s\n", e.what());
```

Comment:
Should this be printed to `stderr`?

Reply:
All other logs are printed already to stdout

Reply:
There is a whole bunch that print to stderr, is there another reason not to? Do you have a preference? Should more be printed to stderr?

Reply:
Ok, i don't mind. In this pr or another one?

Reply:
I don't mind either.